stats.cpp

```diff
@@ -1,5 +1,20 @@
 #include "stats.h"
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Stats Statistics::ComputeStatistics(const std::vector<float>& v) {
```

Comment:
`Stats` is inside a namespace in the header file. Perhaps that's why the compiler complains on this line?